packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -132,12 +140,26 @@ const useSandbox = (
     [onError, fieldPluginURL],
   )
 
+  const dispatchLoadedChanged = useCallback(
+    (message: LoadedMessage) => {
+      postToPlugin(message)
+    },
+    [postToPlugin],
+  )
   const dispatchStateChanged = useCallback(
     (message: StateChangedMessage) => {
       postToPlugin(message)
     },
     [postToPlugin],
   )
+  useEffect(() => {
+    if (!subscribeState) {
+      return
+    }
+
+    dispatchStateChanged(stateChangedData)
+  }, [dispatchStateChanged, stateChangedData])
```

Comment:
The same as the previous comment:

![image](https://github.com/storyblok/field-plugin/assets/1240591/ef41ff15-dde8-4dcb-bf6e-fea3d6973677)